File: C_Programming/Day_4/program21.c
//Description : Write a program that accepts number from the user & count the 
//              frequency of the digit

#include<stdio.h>

int CountFrequency(int iNo , int iTarget)
{
    int iDigit = 0;
    int iCount = 0;

    while(iNo != 0)
    {
        iDigit = iNo % 10;
        if(iDigit == iTarget)
        {
            iCount++;
        }
        
        iNo = iNo / 10;
    }

    return iCount;

}

int main()
{
    int iValue = 0;
    int iRet = 0;
    int iTarget = 0;

    printf("Enter the number : ");
    scanf("%d",&iValue);

    printf("Enter the target : ");
    scanf("%d",&iTarget);

    iRet = CountFrequency(iValue , iTarget);

    printf("Frequency is  : %d\n",iRet);

    return 0;
}

File: Python_Programming/Day_4/program21.py
# Description : Write a program tha accepts number from the user & count the 
#               frequency of the digit

def CountFrequency(No , Target):
    Digit = 0
    Count = 0

    while(No != 0):
        Digit = No % 10
        if(Digit == Target):
            Count += 1
        No = No // 10

    return Count

def main():

    Value = 0
    Ret = 0
    Target = 0

    print("Enter the number : ")
    Value = int(input())

    print("Enter the target : ")
    Target = int(input())

    Ret = CountFrequency(Value , Target)

    print("Frequency is : ",Ret)

if __name__ == "__main__":
    main()